public/locales/it/translation.json

```diff
@@ -2,7 +2,62 @@
     "sign_in": "Accedi",
     "sign_out": "Esci",
     "sign_up": "Registrati",
-    "account": "Profilo",
```

Comment:
A though here. This will most likely break the other translations?

Reply:
Why? Because these translations doesn't have a "parent"? It works, I have already tried